frontend/app/controllers/spree/taxons_controller.rb

```diff
@@ -2,19 +2,36 @@ module Spree
   class TaxonsController < Spree::StoreController
     helper 'spree/products'
 
+    before_action :load_taxon, :load_products
+
     respond_to :html
 
     def show
```

Comment:
Please remove `show` method when nothing is defined in it.

Reply:
@PSzyszkaSpark let's leave it for readability (don't need to lookup routes to see what actions are defined)